### PR TITLE
docs: fix broken link in footer

### PR DESCRIPTION
**Pull request by bokub**
### 🔗 Linked issue
 
Do I really need one?

### ❓ Type of change

- [x] 📖 Documentation (updates to the documentation, readme or JSdoc annotations)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] 🧹 Chore (updates to the build process or auxiliary tools and libraries)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

Fix a broken link found in the footer.
Which one should I use as a replacement ?
- https://www.radix-vue.com
- https://github.com/unovue/radix-vue

### 📸 Screenshots (if appropriate)

This links point to a 404:

image

### 📝 Checklist

- [ ] I have linked an issue or discussion.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: apps/www/.vitepress/theme/layout/MainLayout.vue [hunk 1]
```diff
@@ -209,7 +209,7 @@ watch(() => $route.path, (n) => {
             <span class="inline-block ml-2">
               Ported to Vue by
               <a
-                href="https://github.com/radix-vue"
+                href="https://www.radix-vue.com/"
```
Comment (zernonia) [inline apps/www/.vitepress/theme/layout/MainLayout.vue]:
```suggestion
                href="https://github.com/unovue"
```

Comment (bokub) [reply apps/www/.vitepress/theme/layout/MainLayout.vue]: :heavy_check_mark: Done

**Comment by zernonia** [comment]
Thanks for the PR and changes @bokub !

**Comment by bokub** [comment]
No problem! Thanks for your hard work maintaining this great project! :heart_hands:

**Pull request merged**